airbyte-config/models/src/main/resources/types/SyncStats.yaml

```diff
@@ -0,0 +1,21 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SyncStats.yaml
+title: SyncStats
+description: sync stats.
+type: object
+required:
+  - recordsEmitted
+  - bytesEmitted
+additionalProperties: false
+properties:
+  stream: # if null, represents total stats across all streams
```

Comment:
i have a mild preference towards injecting this information via wrapping it instead of injecting it via a flag. e.g. do something like the object suggested below. And then have `StandardSyncSummary#streamStats` be a list of `StreamSyncStat` instead of generic `SyncStats`
```
  StreamSyncStat:
    type: object
    properties:
      streamName:
        type: string
      stat:
        "$ref": SyncStats.yaml
```
The reason for that is that in the current implementation any reader of this object needs to know what the stream field means even though in some of those cases it isn't relevant. Wrapping it means the concept isn't leaked where it doesn't make sense. To be fair though, the approach I am suggesting  is another yet another wrapper object.

Reply:
Makes sense to me! done.